tests/test_optimizer.py

```diff
@@ -324,6 +324,44 @@ def test_no_iterations():
     np.testing.assert_allclose(x0, out2["x"])
 
 
+@pytest.mark.regression
+@pytest.mark.optimize
+def test_proximal_scalar():
```

Comment:
Do we need a full regression test for this? Can't we just check that the function works?